docs/release-notes.rst

```diff
@@ -36,6 +36,11 @@ Changes and New Features
 
    * - Version
      - Description
+   * - 24.7.0
+     - | - Added support for OpenShift Container Platform v4.16.
```

Comment:
@e0ne 
add:
Support for RHEL with containerd and DOCA-OFED container deployment
Support Switchdev SRIOV mode with OVS CNI - Externally managed PF/Bridge - Testing only
Switchdev VF config restoration for OVS-CNI in MOFED/DOCA container

Reply:
done